--- dialog-1.1-20120706/makefile.in.whole_program~	2012-12-27 05:17:01.696468108 +0100
+++ dialog-1.1-20120706/makefile.in	2012-12-27 05:18:40.547724793 +0100
@@ -159,7 +159,11 @@ HDRS = \
 	dlg_config.h \
 	dlg_keys.h
 
+ifndef WHOLE_PROGRAM
 ALL		= $(LIB) $(PROG)
+else
+ALL		= $(PROG)
+endif
 
 BIN_DIRS	= $(BINDIR) $(MAN1DIR)
 LIB_DIRS	= $(LIBDIR) $(INCLUDEDIR)
@@ -182,9 +186,16 @@ $(LIB) : $(LIB_OBJECT)
 	$(LIBTOOL_CREATE) $(LIB) $(LIB_OBJECT)
 	$(RANLIB) $@
 
+ifndef WHOLE_PROGRAM
 $(PROG)$x : $(LIB) dialog$o @INTLDIR_MAKE@ @INTLLIBS@
 	$(LINK) -o $@ dialog$o -L. -l@PACKAGE@ $(LDFLAGS) $(LIBS)
 
+else
+SRCS_PATH := $(patsubst %,$(srcdir)/%,$(SRCS) dialog.c)
+$(PROG)$x :$(srcdir)/dialog.h $(srcdir)/dlg_keys.h dlg_config.h VERSION  @INTLDIR_MAKE@ @INTLLIBS@
+	$(CC) -o $@ $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -Wl,--no-warn-common -flto -fwhole-program $(SRCS_PATH) $(LIBS)
+endif
+
 clean	\
 distclean \
 install \
